dataland-frontend/src/components/wrapper/AuthorizationWrapper.vue

```diff
@@ -24,13 +29,16 @@ export default defineComponent({
   data() {
     return {
       hasUserRequiredRole: null as boolean | null,
+      isUserDataOwner: null as boolean | null,
+      waitingForData: true,
```

Comment:
I suggest we call this `waitingForDataOwnershipData`.

Otherwise it is not clear to other readers, why we are waiting for any data in the AuthorizationWrapper.

I also needed some minutes until I got it.

Reply:
Renamed